data/json/recipes/food/unshell.json

```diff
@@ -0,0 +1,120 @@
+[
+  {
+    "type": "recipe",
+    "result": "pistachio_unshelled",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "skills_required": [ "survival", 0 ],
```

Comment:
Don't think it changes anything to specify needed skill level of 0; could just remove it.

Reply:
I wanted it to unlock at 0 but train cooking a bit. I just tested and it doesn't unlock at 0.

Reply:
Ok, removed, also difficulty. Unlocks at 0 now.